app.js

```diff
@@ -607,8 +606,17 @@ define(function(require) {
 
 			template.find('.account-settings-update').on('click', function() {
 				if (monster.ui.valid(template.find('#account_settings_form'))) {
+
 					var formData = monster.ui.getFormData('account_settings_form'),
-						newData = $.extend(true, {}, data.account, formData);
+						newData = $.extend(true, {}, data.account, formData),
+						callerIdNumber = _.get(newData, 'caller_id.internal.number') ? monster.util.getFormatPhoneNumber(_.get(newData, 'caller_id.internal.number')) : false;
```

Comment:
Use `lodash#has` instead of `lodash#get` to check for property existence. Also, when ternaries get long, you should break them down for better readability.
```suggestion
						callerIdNumber = _.has(newData, 'caller_id.internal.number')
							? monster.util.getFormatPhoneNumber(_.get(newData, 'caller_id.internal.number'))
							: false;
```
Also, you could make your falsy value an empty object so you would only need to check the for `e164Number` in your `if` later.